reviews two/swagger.yaml

```diff
@@ -0,0 +1,879 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: Draft Orders API
+  description: >
+    ### Draft Orders API 
+      
+      This API is about operations you can perform on the draft order object as defind below:
+    
+    Field | Definition | Data Type
+      -----|-----------|-------------------------
+      `id`| `id` of the draft order |`string`
+      `order_id`  | The id of the order associated to the draft order, once created. | `string`        
+      `name` | Name of the draft order, format #D<number>, where number, is an sequential identifier unique to the shop, starting at 1. For example #D133 | `integer`
+      `customer` | Customer object will be serialized with only the default address, however only the ID can be set in order to associate the customer to the draft order. Setting the value to null removes the customer from the draft order. | `object`
```

Comment:
This is an internal description. Change to something like (my english isn't perfect so rewrite in any way!):

The customer associated with this draft order. The customer is read only, however a new customer can be specified by replacing the ID. Setting the value to null removes the customer from the draft order.